recipes-kernel/linux/nilrt-nohz/README.nohz

```diff
@@ -0,0 +1,98 @@
+==============================
+NI Linux Real-Time NOHZ Kernel
+==============================
+
+This document describes the recommended application design, configuration
+options and boot parameters required for obtaining increased real-time (RT)
+performance when using the NILRT nohz kernel.
+
+The nohz kernel provided by 'packagegroup-ni-nohz-kernel' is a Linux kernel
+configured to omit scheduling-clock interrupts on CPUs that are either idle or
+that have only one runnable task (CONFIG_NO_HZ_FULL=y). This avoids the OS
+induced jitter caused by processing scheduling interrupts at the cost of
```

Comment:
Either "OS-induced" or maybe omit "induced?" The [kernel documentation](https://www.kernel.org/doc/html/latest/timers/no_hz.html) uses the phrase "OS jitter."